akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -148,6 +148,13 @@ akka.cluster.sharding {
     number-of-entities = 5
   }
 
+  event-sourced-remember-entities-store {
+    # When using remember entities and the event sourced remember entities store the batches
+    # written to the store are limited by this number to avoid getting a too large event for
+    # the journal to handle.
+    max-updates-per-write = 500
```

Comment:
I took this number out of thin air, any ideas what would be reasonable to not hit max atomic writes/limits in the most common journals?

Reply:
@chbatey @patriknw

Reply:
Shall we go with 100 assuming a 50 byte key? That is a large key. The C* plugin will warn at a batch over 5kb and drop the write at a request over 50kb with the default settings